### PR TITLE
Added aria-label for button

### PR DESCRIPTION
**Pull request by pokharelmanish**
## JIRA
No story

## Description
Added aria-label for accessibility. 

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters for the project.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/common/Ellipsis.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default class Ellipsis extends React.Component {
     const { id, clientId } = this.props
     return (
       <div>
-        <Button id={`icon-${id}`} onClick={this.toggle} className="icon-ellipsis">
+        <Button id={`icon-${id}`} className="icon-ellipsis" aria-label="button" onClick={this.toggle}>
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/common/Ellipsis.js]: I'm sure users will find it incredibly helpful when screen readers announce that this button has a label of 'button.' :-) It should be something human readable, like "Action Menu" or "Ellipsis Menu Button".

Comment (pokharelmanish) [reply app/javascript/Application/components/common/Ellipsis.js]: sure

Comment (NagaPradeepGajula-osi) [reply app/javascript/Application/components/common/Ellipsis.js]: +1

**Comment by pokharelmanish** [comment]
retest this please

**Pull request merged**